gfi/cache.py

```
@@ -0,0 +1,186 @@
+
+        cached = {
+            'timestamp': datetime.now().isoformat(),
+            'data': value
+        }
+
+        try:
+            with open(cache_path, 'w') as f:
+                json.dump(cached, f, indent=2, default=str)
+        except Exception:
+            # Silently fail on cache write errors
+            pass
```

Ruff (0.15.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)